src/Bicep.Core/Configuration/bicepconfig.json

```diff
@@ -61,5 +61,6 @@
         }
       }
     }
-  }
+  },
+  "experimentalFeaturesEnabled": {}

```

Comment:
>experimentalFeaturesEnabled [](http://example.com/codeflow?start=3&length=27)

We have a JSON schema for the configuration. Should we add the new property and sub-properties to it?

Reply:
I added this to schema file but didn't document the sub-properties. We don't document the environment variables today, and it seemed like a lot of content to generate 😅